docs/components/checkbox.md

```
@@ -4,11 +4,21 @@ import FwbCheckboxExampleDisabled from './checkbox/examples/FwbCheckboxExampleDi
 import FwbCheckboxExampleGroup from './checkbox/examples/FwbCheckboxExampleGroup.vue'
 import FwbCheckboxExampleHelper from './checkbox/examples/FwbCheckboxExampleHelper.vue'
 import FwbCheckboxExampleLink from './checkbox/examples/FwbCheckboxExampleLink.vue'
+import FwbCheckboxExampleStyling from './checkbox/examples/FwbCheckboxExampleStyling.vue'
+import FwbCheckboxExampleValidation from './checkbox/examples/FwbCheckboxExampleValidation.vue'
 </script>
 
 # Vue Checkbox - Flowbite
 
-## Checkbox example
+#### Use the checkbox component to allow users to select one or more options from a set, with support for validation, helper text, and accessible label linking
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)